libmscore/score.h

```diff
@@ -541,6 +542,8 @@ class Score : public QObject, public ScoreElement {
       Score(MasterScore*, const MStyle&);
       virtual ~Score();
 
+      static bool& isScoreLoaded() { return _loadScore; }
```

Comment:
Not sure, but I thing a getter (`isScoreLoaded()`) should go along with a setter (`scoreLoaded(bool b=true)`) and not be the same thing

Reply:
It is not the getter, it is some kind of singleton. It is correct solution to prevent using extern global variables. It cannot be replaced with get/set methods, because static variables lives and visible in the method itself, not in any other scope.

Reply:
Ah, I see

Reply:
Well, now I don't understand. Current implementation is based on get/set logic actually. I thought @handrok made the local static variable. @handrok, could you replace class-scope static variable with function-scope one? Otherwise you need to introduce getter and setter.